spec/acceptance/candlepin_spec.rb

```diff
@@ -1,11 +1,25 @@
 require 'spec_helper_acceptance'
 
 describe 'certs' do
+  FQDN = fact('fqdn')
```

Comment:
This is generating warnings: you can't redefine constants.

Reply:
Those warnings already exist because of this pattern in the other test files. So I'd prefer to keep this pattern and then fix it later across all the test files.

One a side note, it sucks this constant is not localized to the describe block as one might think.

Reply:
Agreed, but that's rspec for you.

Reply:
Proper fix -- https://github.com/theforeman/puppet-certs/pull/372